docs/source/use_cases/run_picard_and_gatk/index.rst

```diff
@@ -143,7 +143,7 @@ Timing the reading speed from the cloud
 You can run `gatk-tools-java/src/main/scripts/example.sh <https://github.com/googlegenomics/gatk-tools-java/blob/master/src/main/scripts/example.sh>`_ with and without "grpc" command line parameter to see the difference in reading speed. The timing statistics are dumped to the terminal.
 We benchmarked **x11** speed improvements with GRPC compared to REST, giving **~12,000 reads/second**.
 
-The tests were done on `Platinum Genomes NA12877_S1.bam dataset <https://console.developers.google.com/storage/browser/genomics-public-data/platinum-genomes/bam/?_ga=1.197206447.160385476.1431305548>`_, please see the `detailed writeup of the test procedure and results <https://docs.google.com/document/d/1Br7RMSbAChNpG6pi2teujf-YthczF-rAM1afSgDoCgQ/edit#>`_ if you want to repeat the test.
+The tests were done on `Platinum Genomes NA12877_S1.bam dataset <https://console.cloud.google.com/storage/browser/genomics-public-data/platinum-genomes/bam/?_ga=1.197206447.160385476.1431305548>`_, please see the `detailed writeup of the test procedure and results <https://docs.google.com/document/d/1Br7RMSbAChNpG6pi2teujf-YthczF-rAM1afSgDoCgQ/edit#>`_ if you want to repeat the test.
```

Comment:
The Google Doc requires access permission, is that a prerequisite or should it be public?